delfin/drivers/netapp/dataontap/constants.py

```diff
@@ -0,0 +1,390 @@
+# Copyright 2021 The SODA Authors.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+from delfin.common import constants
+
+CLUSTER_SHOW_COMMAND = "cluster identity show"
+VERSION_SHOW_COMMAND = "version"
+STORAGE_STATUS_COMMAND = "system health status show"
+
+POOLS_SHOW_DETAIL_COMMAND = "storage pool show -instance"
+AGGREGATE_SHOW_DETAIL_COMMAND = "storage aggregate show -instance"
+POOLS_SPLIT_STR = "   Storage Pool N"
+AGGREGATE_SPLIT_STR = "                                       Aggregat"
```

Comment:
Because the number of Spaces is fixed, if this changed, a single space plus STR will also be used as the split identifier, An object may have both " str" and "          str"，This leads to inaccurate data separation.

Reply:
example : " str" and "       str"

Reply:
Not clear

Reply:
This means that the space of the split string needs to be down to one bit, otherwise the object will be split into two or more objects